src/v0/sources/shopify/transform.js

```diff
@@ -152,6 +152,12 @@ const processEvent = (inputEvent) => {
     version: '1.0.0',
   });
   message.setProperty('context.topic', shopifyTopic);
+
+  // attaching cart, checkout and order tokens in context object
+  message.setProperty(`context.cart_token`, event.cart_token);
+  message.setProperty(`context.checkout_token`, event.checkout_token);
+  message.setProperty(`context.order_token`, event.order_token);
```

Comment:
Don't we need to update these attributes in camel case format?

Reply:
I kept it in similar format as coming from Shopify, so that it is distinguishable from RudderStack spec properties.

Reply:
Got it.